src/main/java/org/skriptlang/reflect/java/elements/structures/StructImport.java

```diff
@@ -45,8 +45,7 @@ public class StructImport extends Structure {
 
     // TODO try replacing ImportHandler with JavaType's literal parsing
     Skript.registerExpression(ImportHandler.class, JavaType.class, ExpressionType.SIMPLE);
-    thisInfo = StreamSupport.stream(
-            Spliterators.spliteratorUnknownSize(Skript.getExpressions(), Spliterator.ORDERED), false)
+    thisInfo = StreamSupport.stream(Spliterators.spliteratorUnknownSize(Skript.getExpressions(), Spliterator.ORDERED), false)
         .filter(i -> i.c == ImportHandler.class)
```

Comment:
Do you think we can replace some of these single-letter variable names as we go through? It's impossible to tell what this is without opening it in an IDE.